rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/MuteExecutionTest.kt

```diff
@@ -256,6 +275,97 @@ open class MuteExecutionTest : AbstractTest() {
+    }
+
+    @Test
+    @Ignore
+    // TODO ("Fix: Mute annotation at line 33 failed - ELEM = "5" - Left value 0  - right value 5 - Line 34")
+    fun executeMUTE15_04() {
+        executePgm("mute/MUTE15_04", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    @Ignore
+    // TODO ("Fix: Issue executing ExecuteSubroutine at line 17. Program MUTE15_05 - Issue executing EvalStmt at line 27. Program <UNSPECIFIED> - Issue executing CallStmt at line 48. Data definition XXSTR was not found")
+    fun executeMUTE15_05() {
+        executePgm("mute/MUTE15_05", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_06() {
+        executePgm("mute/MUTE15_06", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_07() {
+        executePgm("mute/MUTE15_07", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_08() {
+        executePgm("mute/MUTE15_08", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_09() {
+        executePgm("mute/MUTE15_09", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    @Ignore
+    // TODO ("Fix: Issue executing IfStmt at line 416. Program JAJAX0 - Issue executing ClearStmt at line 417. I do not know how to clear ArrayAccessExpr(array=DataRefExpr(variable=Ref(£JSP)[Solved], position=Position(start=Line 417, Column 25, end=Line 417, Column 82)), index=DataRefExpr(variable=Ref(Xml_Liv)[Solved], position=Position(start=Line 417, Column 29, end=Line 417, Column 37)), position=null))
+    fun executeMUTE15_10() {
+        val configuration = Configuration().apply {
+            // this way I enable muteSupport and source dump also on execution error
+            options = Options(muteSupport = true, dumpSourceOnExecutionError = true)
+        }
+        executePgm("mute/MUTE15_10", configuration = configuration, systemInterface = JavaSystemInterface())
+        // assertCanBeParsedResult(exampleName = "mute/MUTE15_10", printTree = true)
+    }
+
+    @Test
+    fun executeMUTE15_11() {
+        executePgm("mute/MUTE15_11", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_12() {
+        executePgm("mute/MUTE15_12", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_13() {
+        executePgm("mute/MUTE15_13", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    fun executeMUTE15_14() {
+        executePgm("mute/MUTE15_14", configuration = Configuration().apply { options = Options(muteSupport = true) })
+    }
+
+    @Test
+    @Ignore
```

Comment:
Missing todo

Reply:
There is no reason to Ignore MUTE15_15 test. Maybe it should be enabled.